tests/system/src/test/java/test/javafx/embed/swing/SwingNodeMemoryLeakTest.java

```diff
@@ -77,8 +77,6 @@ public void testSwingNodeMemoryLeak() throws InterruptedException, InvocationTar
             testSwingNodeObjectsInStage();
         });
 
-        SwingUtilities.invokeAndWait(() -> Util.sleep(500));
-
         JMemoryBuddy.assertCollectable(weakRefArrSN);
```

Comment:
On my Mac (MacBook Pro x64 running macOS 13.2.1) I saw a one-time test failure here. So there might be a need to wait for the EDT to at least do something, possibly followed by a sleep (on the test thread, not on the EDT).

Reply:
Interesting, I will add a `runAndWait()` noop and a test thread sleep then.

Reply:
I think you mean `SwingUtilities::invokeAndWait`, right?

Btw, I also got a similar one-time failure in `SwingNodeDnDMemoryLeakTest`, so perhaps the same could be added there?

Reply:
Yes, I meant `invokeAndWait`. A force of habit from using our `Util` class...

I also added it to `SwingNodeDnDMemoryLeakTest` just to be sure. After a few runs it works fine on my end (but also, it did earlier so I can't really fully test it).